cmd/site-replication.go

```diff
@@ -3918,21 +4231,46 @@ func getSRBucketDeleteOp(isSiteReplicated bool) SRBucketDeleteOp {
 	return MarkDelete
 }
 
+func (c *SiteReplicationSys) healILMExpiryConfig(ctx context.Context, objAPI ObjectLayer, info srStatusInfo) error {
+	c.RLock()
+	defer c.RUnlock()
+	if !c.enabled {
+		return nil
+	}
+
+	for dID, pi := range info.Sites {
+		if dID == globalDeploymentID() {
+			continue
+		}
+		admClient, err := c.getAdminClient(ctx, dID)
+		if err != nil {
+			return wrapSRErr(err)
+		}
+		if err = admClient.SRStateEdit(ctx, madmin.SRStateEditReq{Peers: info.Sites, UpdatedAt: info.UpdatedAt}); err != nil {
+			logger.LogIf(ctx, c.annotatePeerErr(pi.Name, siteReplicationEdit,
+				fmt.Errorf("Unable to heal site replication state for peer %s from peer %s : %w",
+					pi.Name, c.state.Name, err)))
+		}
+	}
```

Comment:
you can't blindly copy all settings over - sync/async mode and bandwidth throttling are meant to be unidirectional. Making a change on one site should not be copied over to others. Ensure that healing happens only on the site with the latest update to the remaining on just the ilm setting

Reply:
Sure @poornas , at receiving end I will make sure only `ReplicateILMExpiry` flag is replicated and `UpdatedAt` time get updated. Will change and update.